app/views/posts/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Edit a blog post!</h1>
+
+<%= render "/posts/form" %>
```

Comment:
Could have just been `render "form"` ... I nearly left it alone, though, because specificity of which folder to find the partial in comes in handy sometimes.